addons/customGear/config.hpp

```diff
@@ -2,18 +2,16 @@
 
 class CfgPatches {
     class ADDON {
-        weapons[] = {
-            QGVAR(MG5)
-        };
         units[] = {};
+        weapons[] = {};
```

Comment:
we might want to put weapons in here, 
i don't think it matters for arsenal but I think it does for the inventory box editor in eden/zeus?

Reply:
done!